.circleci/config.yml

```diff
@@ -102,7 +102,8 @@ jobs:
       - checkout
       - restore_cache:
           key: mac-go-mod-v2-{{ checksum "go.sum" }}
-      - run: 'brew install go' # latest
+      - run: 'curl https://raw.githubusercontent.com/Homebrew/homebrew-core/a8c62d0d1e88104a47f51ef911ceb8abf3c7f071/Formula/go.rb --output go.rb'
+      - run: 'brew install go.rb'
```

Comment:
Is this just for now or will this stay?

Reply:
I was thinking this would stay, this was the only way I could figure out to get a specific version of go installed with brew. The command `brew install go@1.16` wasn't working (example: https://app.circleci.com/pipelines/github/influxdata/telegraf/3685/workflows/808b7b42-e3b3-43a0-95a6-d41ba10d7820/jobs/79890).

Reply:
Hmmm I think `go.rb` is 1.16 and as soon as `1.17` will be available you can use `go@1.16`. But I'm not sure... Was just a question anyway.

Reply:
brew tends to be slow with releases. it should be available now.
```
 brew search go@1.16
==> Formulae
go@1.16 ✔
```